src/shims/clipboard-polyfill/index.js

```diff
@@ -0,0 +1,12 @@
+/** This file is an auto-generated shim, aliased in for "clipboard-polyfill" in the webpack config.
+*  When components import 'clipboard-polyfill', we want to give them back the copy
+*  Zimbra passed down when it called the factory provided to zimlet().
+*/
+
+/* eslint-disable camelcase, dot-notation */
+import { warnOnMissingExport } from '../';
+const wrap = warnOnMissingExport.bind(null, global.shims['clipboard-polyfill'], 'clipboard-polyfill');
+
+export const writeText = wrap('writeText');
```

Comment:
I am thinking to remove this library from core, can we use native browser's API if we want to copy just plain text data?

Reply:
we had discussed this point earlier , it was giving some issues when writing the text , so then i used this

Reply:
ok

Reply:
i wont expose the library  from shims , for now i will install in zimlet in future when we remove the library we can change in zimlet , so that we dont have dependency on zm-x-web for this library

Reply:
have reverted the changes